dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/plugins/custom/OauthClientStore.java

```diff
@@ -320,8 +321,8 @@ void logMessageAndThrowOidcServerException(OAuthStoreException e, String msgKey,
 
         // Use an empty string as the last insert to avoid an unused insert (e.g. "{2}") showing up in the exception message
         updatedMsgArgs = appendStringMessageToArgs("", msgArgs);
-        String exceptionMsg = Tr.formatMessage(tc, msgKey, updatedMsgArgs);
-        throw new OidcServerException(exceptionMsg, OIDCConstants.ERROR_SERVER_ERROR, HttpServletResponse.SC_INTERNAL_SERVER_ERROR, e);
+        throw new OidcServerException(new BrowserAndServerLogMessage(tc, msgKey, exceptionMsg), OIDCConstants.ERROR_SERVER_ERROR, HttpServletResponse.SC_INTERNAL_SERVER_ERROR, e);
```

Comment:
Think you meant to include `updatedMsgArgs` as the third argument to the `BrowserAndServerLogMessage` constructor, not `exceptionMsg`.